### DISCUSSION TITLE
How to bind Ctrl-Semicolon?

### DISCUSSION THREAD

**Discussion by cd-a** [helix-editor/helix#9125] [Q&A] [answered]
I can't figure out how to map C-; properly.

Writing it like this does not work.
```toml
"C-;" = "vsplit"
```

I know those maps are supported in helix, as `A-;` exists.

**Comment by the-mikedavis** [comment, marked as answer]
That binding works for me on Kitty. It looks like without the enhanced keyboard protocol (https://github.com/helix-editor/helix/wiki/Terminal-Support#enhanced-keyboard-protocol), the terminal sends `;` when you press `C-;`

**Reply by cd-a** [reply]
Thanks! After enabling the enhanced protocol in wezterm it works like a charm.
